website/content/api-docs/secret/pki.mdx

```diff
@@ -1314,7 +1314,22 @@ $ curl \
 ```json
 {
   "data": {
-    "certificate": "-----BEGIN CERTIFICATE-----\nMIIGmDCCBYCgAwIBAgIHBzEB3fTzhTANBgkqhkiG9w0BAQsFADCBjDELMAkGA1UE\n..."
+    "certificate": "-----BEGIN CERTIFICATE-----\nMIIGmDCCBYCgAwIBAgIHBzEB3fTzhTANBgkqhkiG9w0BAQsFADCBjDELMAkGA1UE\n...",
+    "revocation_time": 0
+  }
+}
+```
+
+As of Vault 1.13.0, a revoked certificate may contain two additional fields, `revocation_time_rfc3339` and `issuer_id`
```

Comment:
I don't think we really need this; it is just meant to be representative and we've only historically really documented request parameters vs response parameters: https://developer.hashicorp.com/vault/api-docs 

Maybe just use the second example rather than the former?

Reply:
Fair, updated with second example